src/ansys/dpf/composites/data_sources.py

```diff
@@ -38,13 +39,43 @@ class CompositeDefinitionFiles:
 class ContinuousFiberCompositesFiles:
     """Provides the container for continuous fiber composite file paths."""
 
-    rst: _PATH
+    rst: List[_PATH]
```

Comment:
Can we rename this parameter to rsts or result_files?
rst is confusing because it is a list. And we plan to support LS Dyna or thermal analyses (rth) as well.

Reply:
Hmm, to my eye `rst` is consistent with the `composite` attribute that also refers to multiple composite definition files.

For LS-Dyna and RTH files: Do you know if we'll need to explicitly distinguish between them, or can just use the `DataSources` interface on these to abstract away the differences? If we need to treat them explicitly, they probably need a separate attribute.